src/SiteExtensions/LoggingBranch/LoggingBranch.csproj

```diff
@@ -23,11 +24,23 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <HostingStartupRuntimeStoreTargets Include="netcoreapp2.2" Runtime="win7-x64" />
-    <HostingStartupRuntimeStoreTargets Include="netcoreapp2.2" Runtime="win7-x86" />
+    <HostingStartupRuntimeStoreTargets Include="netcoreapp3.0" Runtime="$(SharedFxRid)" />
+
     <HostingStartupPackageReference Include="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Version="$(PackageVersion)" />
 
     <PackageReference Include="Internal.AspNetCore.Sdk" PrivateAssets="All" Version="$(InternalAspNetCoreSdkPackageVersion)" />
+
+    <ProjectReference Include="..\..\Framework\Microsoft.AspNetCore.App\pkg\Microsoft.AspNetCore.App.pkgproj">
```

Comment:
Do you also need to reference Microsoft.AspNetCore.All?

Reply:
It trims against `.App` so this should be enough.